packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/proxy.integtest.ts

```diff
@@ -65,7 +65,6 @@ async function runInIsolatedContainer(fixture: TestFixture, pathsToMount: string
       'public.ecr.aws/ubuntu/ubuntu:24.04_stable',
       `${scriptName}`,
     ], {
-      stdio: 'inherit',
```

Comment:
Removed because this causes `child.stdout` to be null. The reason is used to work is because we were overriding this configuration:

https://github.com/aws/aws-cdk-cli-testing/blob/098e373ed66c609b5869e97acec199fa0d6f08e0/packages/%40aws-cdk-testing/cli-integ/lib/shell.ts#L30-L36

We don't anymore. This was probably unintended because we still want test to be able to configure it if needed.